### PR TITLE
Added readme section how to allow usb permission for python as unprivileged user.

### PR DESCRIPTION
**Pull request by aurelijusrozenas**
This works for me, should be useful for everyone :)

### CODE REVIEW

**Comment by vmitchell85** [comment]
Thanks for this... sorry I didn't see it before now. Would you mind reformatting the file creation into a code snippet so it's easier to copy and paste?

**Comment by aurelijusrozenas** [comment]
How about now? ;)

File: readme.md [hunk 1]
```diff
@@ -30,12 +30,16 @@ Credit to gist by [dave-irvine](https://gist.github.com/dave-irvine/dbec2584e750
 ### Linux
 - Sometimes first run doesn't work, have to run command a second time
 
-### Allow usb permissions for python in Ubuntu 16.04
+### Allow usb permissions for python in Ubuntu 16.04/16.10
 By default python will not let use usb libs and you will need to run command with `sudo` everythime. Let's no do that.
```
Comment (vmitchell85) [inline readme.md:34]: Typo... "Lets no do that"

**Comment by vmitchell85** [comment]
I'm not really a fan of running a single command to echo out a file to the user's filesystem. I'd prefer instructions on what file needs to be created and the code snippet for the actual file. Hope that makes sense. If you'd prefer not to die it I can make the necessary changes, just let me know.

Not trying to be picky, just want to make sure it's easy to know what to do if you're not an advanced linux user and want to get it working.

Thanks,
- Vince

**Comment by aurelijusrozenas** [comment]
If you have something in mind how it should look, feel free to copy my text and commit you version of it. I rewrote text and snippet several times but could not find one clear and easy variant. And be picky, it's better for everyone :)

**Comment by vmitchell85** [comment]
See #5

**Pull request closed without merging**